lf-queue/Makefile

```diff
@@ -1,14 +1,17 @@
-CFLAGS =  -Wall -O2 -g -I.
-LDFLAGS = -lpthread
+all: 
```

Comment:
Check https://github.com/sysprog21/jitboy/blob/master/Makefile for modularized build rules.

Reply:
Done!